tests/components/universal/test_media_player.py

```diff
@@ -681,6 +681,184 @@ def test_supported_features_children_and_cmds(self):
 
         assert check_flags == ump.supported_features
 
+    async def test_overrides(self):
```

Comment:
I think the default unittest TestCase class doesn't support coroutine functions as test methods. The coroutine is probably not awaited.